src/us/kbase/workspace/kbase/LocalTypeServerMethods.java

```diff
@@ -0,0 +1,323 @@
+package us.kbase.workspace.kbase;
```

Comment:
This PR is effectively just moving all the code from `WorkspaceServer` that deals with the type database in here, almost verbatim, plus the two methods from `WorkspaceServerMethods`.